pages/linux/debootstrap.md

```diff
@@ -0,0 +1,16 @@
+# debootstrap
+
+> Create a basic Debian system.
+> More information: <https://wiki.debian.org/Debootstrap>.
+
+- Create a Debian buster system inside {{buster-chroot}} directory:
+
+`sudo debootstrap buster {{path/to/buster-chroot/}} http://deb.debian.org/debian`
+
+- Create an Ubuntu 20.04 system inside {{focal-root}} directory with local mirror:
+
+`sudo debootstrap focal {{path/to/focal-root/}} {{file:///path/to/mirror/}}
```

Comment:
Missing backtick at the end here?
```suggestion
`sudo debootstrap focal {{path/to/focal-root/}} {{file:///path/to/mirror/}}`
```

Also, it would be nice if these examples could be more generic, as these example distribution codenames will be out of date in a few years time.

Reply:
@sbrl 
Thank you, my fault.

I updated the Debian example to be more generic, but I see no such possibility for Ubuntu release. Therefore there are support of every Ubuntu release and command will work forever with local mirror as in the example.